c2rust-analyze/src/analyze.rs

```diff
@@ -425,7 +425,7 @@ fn parse_def_id(s: &str) -> Result<DefId, String> {
     };
 
     let rendered = format!("{:?}", def_id);
-    if &rendered != s {
+    if &rendered != orig_s {
```

Comment:
Was this a bug previously?

Reply:
Yes - it would erroneously reject some `DefId`s when parsing the `--fixed-defs-list` file.  I edited the PR description (which will be included in the eventual merge commit) to mention this